api/nvidia.com/resource/v1beta1/computedomain.go

```diff
@@ -61,6 +61,10 @@ type ComputeDomainSpec struct {
 // ComputeDomainChannelSpec provides the spec for a channel used to run a workload inside a ComputeDomain.
 type ComputeDomainChannelSpec struct {
 	ResourceClaimTemplate ComputeDomainResourceClaimTemplate `json:"resourceClaimTemplate"`
+	// Request all IMEX channels instead of one.
+	// Not required, if omitted translates to `false`.
+	//+kubebuilder:validation:Optional,
+	All bool `json:"all,omitempty"`
```

Comment:
What if we changed this to a `AllocationMode` with (currently) two values `SingleChannel` and `AllChannels` (where `SingleChannel` is the default)?

Reply:
I like
```
  channel:
    resourceClaimTemplate: <name>
    AllocationMode: All
```
better than
```
  channel:
    resourceClaimTemplate: <name>
    AllocationMode: AllChannels
```

Reply:
And thanks for thinking ahead towards representing the isolation mode, too -- we should make that really minimal and expressive, and the best way to do that is by collecting ideas early, and to iterate for a while and give it a bit of time.